docs/features/ethernet-lan.md

```diff
@@ -39,5 +40,22 @@ ESP32Deux |  | 17, 18, 23
 [WESP32](https://wesp32.com/) |  | 0, 16, 17
 [WT32-ETH01](https://www.seeedstudio.com/Ethernet-module-based-on-ESP32-series-WT32-ETH01-p-4736.html) |  | 0, 16, 18, 23
 
+## Defining GPIO for a custom PCBA
```

Comment:
Misleading or incorrectly worded heading.
IMO it should read: **Creating custom/new Ethernet board option in WLED**